lib/LoginStrategy.types.ts

```diff
@@ -21,7 +21,7 @@ export interface DecodedToken {
   sub: string;
 }
 
-export interface ITokenPayload extends DecodedToken {
+export interface ITokenPayload {
```

Comment:
Can we remove the `DecodedToken` interface now? Or is it still used?

Reply:
No it's still used as a type for decoded `claim.issuedToken`